src/style.css

```diff
@@ -0,0 +1,11 @@
+@import url("/card.css");
+@import url("/stars.css");
+
+/* #region Normalize */
+* {
+  margin: 0;
+  display: flex;
+  box-sizing: border-box;
+}
```

Comment:
```suggestion
* {
  box-sizing: border-box;
}
```

Reply:
Hi Virass, I understand the last one, but how to fix the first one. I have everything displayed normally. font face I have not prescribed anywhere at all. 
![image](https://user-images.githubusercontent.com/102994838/189301149-0a5dd84c-8dbc-4558-8df8-515c883184a2.png)